### ISSUE TITLE
Issue with rancher desktop  Error: /Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/bin/limactl exited with code 1

### ISSUE THREAD

**Issue by fromloltocode** [rancher-sandbox/rancher-desktop#2535]
I have an error that shows 
/Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/bin/limactl start --tty=false 0

Context:
Starting virtual machine

Some recent logfile lines:
    at ChildProcess.emit (node:events:390:28)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:290:12) {
  command: [
    '/Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/bin/limactl',
    'start',
    '--tty=false',
    '0'
  ]
}

I've tried the workaround with #1815 and it still doesn't work any ideas? 

I also have the M1 mac chip

**Comment by fromloltocode** [comment]
Also I have the M1 mac chip

**Comment by jandubois** [comment]
There is not enough information to see what is wrong; please attach a zipfile/tarball of the log files (see Troubleshooting page for location).

There is a limitation of 8 CPUs in QEMU on arm64 architecture; make sure you haven't selected more than that for the VM.

**Comment by fromloltocode** [comment]
> 
[log files.zip](https://github.com/rancher-sandbox/rancher-desktop/files/9104405/log.files.zip)

Here are the files although a lot of them are blank so I do not know if they would be much help.

**Comment by jandubois** [comment]
> Here are the files although a lot of them are blank so I do not know if they would be much help.

Indeed, there isn't even the error message from the dialog in the log files, which makes me wonder if the logs are from the same run.

There is also no `lima.ha.stderr.log`. Does that not exist, or is it not included because it is a symlink?

**Comment by jandubois** [comment]
That sounds rather wrong; it should look like this:

```console
$ ls -l ~/Library/Logs/rancher-desktop
total 104
-rw-------  1 jan  staff   3199 13 Jul 14:46 background.log
-rw-------  1 jan  staff      0 13 Jul 14:46 images.log
-rw-------  1 jan  staff      0 13 Jul 14:46 integrations.log
-rw-------  1 jan  staff    193 13 Jul 14:46 k8s.log
lrwxr-xr-x  1 jan  staff     62 17 Jun 09:55 lima.ha.stderr.log -> ../../Application Support/rancher-desktop/lima/0/ha.stderr.log
lrwxr-xr-x  1 jan  staff     62 17 Jun 09:55 lima.ha.stdout.log -> ../../Application Support/rancher-desktop/lima/0/ha.stdout.log
-rw-------  1 jan  staff  20683 13 Jul 14:47 lima.log
lrwxr-xr-x  1 jan  staff     59 17 Jun 09:55 lima.serial.log -> ../../Application Support/rancher-desktop/lima/0/serial.log
-rw-------  1 jan  staff      0 13 Jul 14:46 mock.log
-rw-------  1 jan  staff  16152 13 Jul 14:47 networking.log
-rw-------  1 jan  staff    110 13 Jul 14:46 server.log
-rw-------  1 jan  staff      0 13 Jul 14:46 settings.log
-rw-------  1 jan  staff      0 13 Jul 14:46 steve.log
-rw-------  1 jan  staff      0 13 Jul 14:46 update.log
-rw-------  1 jan  staff      0 13 Jul 14:46 wsl.log
```

I guess we don't have to look into this, if everything is working for you, but it definitely does not sound like the installation is proper.

**Comment by fromloltocode** [comment]
sc-rtran-osx@AZ-L-JQ34Y-OSX log files % ls -l
total 24
-rw-------  1 sc-rtran-osx  staff     0 Jul 13 11:59 background.log
-rw-------  1 sc-rtran-osx  staff     0 Jul 13 11:59 images.log
-rw-------  1 sc-rtran-osx  staff     0 Jul 13 11:59 integrations.log
-rw-------  1 sc-rtran-osx  staff     0 Jul 13 11:59 k8s.log
-rw-------  1 sc-rtran-osx  staff     0 Jul 13 11:59 lima.log
-rw-------  1 sc-rtran-osx  staff     0 Jul 13 11:59 mock.log
-rw-------  1 sc-rtran-osx  staff  6536 Jul 13 11:59 networking.log
-rw-------  1 sc-rtran-osx  staff     0 Jul 13 11:59 server.log
-rw-------  1 sc-rtran-osx  staff     0 Jul 13 11:59 settings.log
-rw-------  1 sc-rtran-osx  staff     0 Jul 13 11:59 steve.log
-rw-------  1 sc-rtran-osx  staff   495 Jul 13 16:35 update.log
-rw-------  1 sc-rtran-osx  staff     0 Jul 13 11:59 wsl.log

This is what I'm getting. Seems like it's missing what you have. I'm not sure what went wrong with the installation. Is there a fix for this?

**Comment by jandubois** [comment]
This was filed against RD 1.4.1, and the discussion stalled after it became clear the Lima VM log symlinks were missing entirely — indicating the VM was never initialized rather than a code bug. The version is long superseded, and the VM backend has changed substantially since (VZ is now default on macOS). If you still see a `limactl exited with code 1` failure on a current release, please open a new issue with logs from the [Troubleshooting page](https://docs.rancherdesktop.io/ui/troubleshooting).
